hub/policy/src/main/java/uk/gov/ida/hub/policy/factories/SessionStoreFactory.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.ida.hub.policy.factories;
+
+import org.joda.time.DateTime;
+import uk.gov.ida.hub.policy.controllogic.TransitionStore;
+import uk.gov.ida.hub.policy.domain.SessionId;
+import uk.gov.ida.hub.policy.domain.State;
+import uk.gov.ida.shared.dropwizard.infinispan.util.InfinispanCacheManager;
+
+import java.util.Optional;
+import java.util.concurrent.ConcurrentMap;
+
+public class SessionStoreFactory {
+
+    private SessionStoreFactory() {}
+
+    public static TransitionStore<State> getSessionStateStore(
+            InfinispanCacheManager infinispanCacheManager
+    ) {
+        return getSessionStore(infinispanCacheManager, "state_cache");
+    }
+
+    public static ConcurrentMap<SessionId, DateTime> getSessionExpirationStore(
```

Comment:
getSessionExpirationStore() method is not used anywhere in policy. Is the method needed in the future?